src/contentmanagerdelegate.cpp

```diff
@@ -182,7 +182,25 @@ void ContentManagerDelegate::paint(QPainter *painter, const QStyleOptionViewItem
             showDownloadProgress(painter, r, *downloadState);
         }
         else {
-            baseButton->style()->drawControl( QStyle::CE_PushButton, &button, painter, baseButton.data());
+            if (option.state & QStyle::State_MouseOver) {
+                baseButton->setStyleSheet("background-color: #eaecf0;"
+                                          "border: 0;"
+                                          "font-weight: bold;"
+                                          "font-family: Selawik;"
+                                          "border-bottom: 1px solid #cccccc;"
+                                          "color: blue;"
+                                          "margin: 0;");
+                baseButton->style()->drawControl( QStyle::CE_PushButton, &button, painter, baseButton.data());
+            }else{
+                baseButton->setStyleSheet("background-color: white;"
+                                          "border: 0;"
+                                          "font-weight: bold;"
+                                          "font-family: Selawik;"
+                                          "border-bottom: 1px solid #cccccc;"
+                                          "color: blue;"
+                                          "margin: 0;");
+                baseButton->style()->drawControl( QStyle::CE_PushButton, &button, painter, baseButton.data());
+            }
```

Comment:
Too much code duplication between the two branches. At least the `drawControl()` line can be moved out.

Then the only difference between the stylesheet strings is the value of background colour. The question is if you had to explicitly specify the other parameters (using their implicitly default values in this context) just because it is impossible to separately set just the background colour.

In any case if the stylesheet string must be provided fully its common part should be separated into a constant.

Reply:
@veloman-yunkan  thanks for your suggestions, implemented them in the latest commit.